modules/Sources/Features/SendFlow/SendFlowStore.swift

```diff
@@ -90,7 +91,10 @@ public struct SendFlowReducer: Reducer {
         }
         
         public var isMemoInputEnabled: Bool {
-            transactionAddressInputState.textFieldState.text.data.isEmpty ||
+            print(transactionAddressInputState.textFieldState.text.data.isEmpty ||
+                  !transactionAddressInputState.isValidTransparentAddress)
+            
+            return transactionAddressInputState.textFieldState.text.data.isEmpty ||
```

Comment:
This looks like a local debugging change that should be undone:
```suggestion
            transactionAddressInputState.textFieldState.text.data.isEmpty ||
```